setup-env.js

```diff
@@ -9,11 +10,12 @@ console.log('\x1b[36m%s\x1b[0m', 'Starting Predator local setup, good luck!'); /
 
 npmInstallAndBuild();
 extractIpAndPort();
-setupEnvFile();
+setupBackendEnvFile();
+setupFrontendEnvFile();
 
 console.log('\x1b[36m%s\x1b[0m', 'To start predator backend + frontend:\nnpm run start-local');
 console.log();
-console.log('\x1b[36m%s\x1b[0m', `To develop frontend with hot reload:\ncd ui\nPREDATOR_URL=http//${ipAddress}:${port}/v1 npm start`);
+console.log('\x1b[36m%s\x1b[0m', 'To develop frontend with hot reload:\ncd ui\nnpm start');
```

Comment:
I don't know if I am exaggerating, should we consider using 
`const { EOL } = require('os');` instead of hardcoding `\n`? this code runs on the user machine and not a controlled environment (our docker image)

Reply:
good point

Reply:
btw are you aware of OS where \n is not a new line?

Reply:
yes windows for example is `\r\n` and macOS is just `\r`

Reply:
good to know ;)